gradlew

```diff
@@ -155,13 +155,14 @@ if $cygwin ; then
 fi
 
 # Escape application args
-for s in "${@}" ; do
-    s=\"$s\"
-    APP_ARGS=$APP_ARGS" "$s
-done
+save () {
+    for i do printf %s\\n "$i" | sed "s/'/'\\\\''/g;1s/^/'/;\$s/\$/' \\\\/" ; done
+    echo " "
+}
```

Comment:
Yeah. I just ran './gradlew wrapper --Gradle-version=4.0' to generate changes in that particular commit. 😅